docs/network-flow-visibility.md

```diff
@@ -289,6 +290,23 @@ ClickHouse cluster can be deployed with default Local PV or NFS PV by setting
 PV creation, you can configure a customized `StorageClass` in
 `clickhouse.storage.persistentVolumeClaimSpec`.
 
+##### Secure Connection
+
+From Theia 0.7, we support TLS connection option in ClickHouse server. To
+enable TLS connection, please set `clickhouse.service.secureConnection.enable`
+to true.
+
+Currently ClickHouse Operator does not support cert-manager, which means users
+need to manually do the certs rotation when using this ClickHouse server TLS
+option. You can set `clickhouse.service.secureConnection.selfSignedCert` to
+true to use the default certs. Please ensure you set the Subject's common name
```

Comment:
```suggestion
true to use the default certs. Please ensure that you have set the subject's common name
```
not sure, do we need to capitalize `subject`? And I think we can remove "and optional IP addresses and DNS names". It is a bit strange to me to say "ensure set ... optional ..."